neo/SmartContract/Contract.cs

```diff
@@ -46,6 +46,20 @@ public static Contract Create(ContractParameterType[] parameterList, byte[] rede
             };
         }
 
+        /// <summary>
+        /// Construct special Contract with empty Script, will get the Script with scriptHash from blockchain when doing the Verify
+        /// verification = snapshot.Contracts.TryGet(hashes[i])?.Script;
+        /// </summary>
+        public static Contract Create(UInt160 scriptHash, params ContractParameterType[] parameterList)
```

Comment:
This is only used in `TransactionManager` maybe should be moved there

Reply:
This method should be here to set the ```_scriptHash```